platform/golden/upload-image.sh

```diff
@@ -0,0 +1,28 @@
+cd build
```

Comment:
comments here and there would be cool - not. really sure what this does without a deep dive into the man pages..